modular_skyrat/modules/customization/modules/surgery/organs/taur_body.dm

```diff
@@ -96,11 +98,26 @@
 	layers = ALL_EXTERNAL_OVERLAYS | EXTERNAL_FRONT_UNDER_CLOTHES | EXTERNAL_FRONT_OVER
 	color_source = ORGAN_COLOR_OVERRIDE
 
+	/// If this taur body can lay down
+	var/can_lay_down = FALSE
+	/// Are we currently laying down?
+	var/laying_down = FALSE
+	/// The offset we get from laying down. Negative values move us down
+	var/laydown_offset = 0
+
+/datum/bodypart_overlay/mutant/taur_body/on_mob_insert(obj/item/organ/parent, mob/living/carbon/receiver)
+	. = ..()
+	var/datum/sprite_accessory/taur/accessory = sprite_datum
+	if(accessory.can_lay_down)
```

Comment:
Challenge: Make it work with resting (If you can manage to change the icon changing resting is easier)